app/src/main/java/com/ataulm/whatsnext/WhatsNextRepository.kt

```diff
@@ -34,13 +37,14 @@ internal class WhatsNextRepository(
         filmSummaries
     }
 
-    suspend fun film(letterboxdId: String) = withContext(Dispatchers.IO) {
+    suspend fun filmSummary(letterboxdId: String) = withContext(Dispatchers.IO) {
```

Comment:
technically, letterboxd has different responses for a film/filmsummary. A `FilmSummary` is usually general info about the film, pretty light, and is returned from other API calls like the one we use for popular films this week.

Even though we're getting a full film back here from the API, we're using the film Summary converter so we're ignoring a bunch of properties, hence the rename for this function.